### PR TITLE
Integrate the operator ui with the kubectl plugin

### PR DESCRIPTION
**Pull request by dvaldivia**
This PR introduces an integration of the Operator UI into 

`kubectl minio init` installs the operator UI resources into the operator namespace
`kubectl minio delete` deletes the same resources that were installed
`kubectl minio proxy` starts a port forward to the console service and shows instructions in how to connect.

Some other changes `make getconsoleuiyaml ` will get the latest operator install from the console repo and re-generate the static assets for the command

Changed the default namespace we install things to `minio-operator` as it's the same namespace we install things via `kubectl -k`

### CODE REVIEW

**Comment by nitisht** [comment]
image

**Comment by nitisht** [comment]
Validation seems wrong in here

**Comment by dvaldivia** [comment]
@nitisht that has to do mostly with a validation error on the console side, I'll send a PR for that separately, it's failing since it expects quotas to exists in the namepace

File: operator-kustomize/console-ui.yaml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      labels:
+        app: console
+    spec:
+      containers:
+      - args:
+        - server
+        env:
+        - name: CONSOLE_OPERATOR_MODE
+          value: "on"
+        image: minio/console:v0.5.0
```
Comment (nitisht) [inline operator-kustomize/console-ui.yaml:161]: Image tag has changed to `minio/console:0.5.0`

Comment (dvaldivia) [reply operator-kustomize/console-ui.yaml:161]: Yes, we made a release, that's the version of console that will work with the current master until the conversion support is merged (then we need another release)

Comment (dvaldivia) [reply operator-kustomize/console-ui.yaml:161]: @nitisht I also fixed the problem you reported when selecting a namespace in that release

Comment (nitisht) [reply operator-kustomize/console-ui.yaml:161]: I meant there is an extraneous `v` in the tag here. Dockerhub tag is `0.5.0`

Comment (dvaldivia) [reply operator-kustomize/console-ui.yaml:161]: Maybe it's intentional so it matches the tags on the console repo? idk hahaha let's ask @harshavardhana

Comment (harshavardhana) [reply operator-kustomize/console-ui.yaml:161]: no we can have `v` from next release onwards @nitisht @dvaldivia I can create an alias for the time being

**Review by nitisht — APPROVED** [review]
Tested & LGTM

File: operator-kustomize/console-ui.yaml (hunk 1 above)
Comment (harshavardhana) [reply operator-kustomize/console-ui.yaml:161]: Or perhaps just leave it as is @dvaldivia we don't need an extra `v` at the end.

**Comment by harshavardhana** [comment]
minio/console:v0.5.0 is available now

File: kubectl-minio/cmd/proxy.go [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+	return cmd
+}
+
+// run initializes local config and installs MinIO Operator to Kubernetes cluster.
+func (o *operatorProxyCmd) run() error {
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// kubectl  get secret $(kubectl get serviceaccount console-sa -o jsonpath="{.secrets[0].name}") -o jsonpath="{.data.token}" | base64 --decode | pbcopy
```
Comment (Alevsk) [inline kubectl-minio/cmd/proxy.go:88]: can this be deleted?

File: kubectl-minio/cmd/proxy.go [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+		if err != nil {
+			log.Printf("cmd.Run() failed with %s\n", err.Error())
+			return
+		}
+		if errStdout != nil || errStderr != nil {
+			log.Printf("failed to capture stdout or stderr\n")
+			return
+		}
+		//outStr, errStr := string(stdout), string(stderr)
+		//fmt.Printf("\nout:\n%s\nerr:\n%s\n", outStr, errStr)
```
Comment (Alevsk) [inline kubectl-minio/cmd/proxy.go:170]: can this be deleted

File: kubectl-minio/cmd/resources/common.go [hunk 4]
```diff
@@ -37,18 +59,131 @@ func Pool(servers, volumes int32, q resource.Quantity, sc string) miniov1.Pool {
...
+					resourceObj.Namespace = opts.Namespace
+				}
+			case *rbacv1.ClusterRoleBinding:
+				if resourceObj, ok := obj.(*rbacv1.ClusterRoleBinding); ok {
+					for _, sub := range resourceObj.Subjects {
+						sub.Namespace = opts.Namespace
+					}
+				}
+			default:
+				// fmt.Println("Unhandled kind:", obj.GetObjectKind())
```
Comment (Alevsk) [inline kubectl-minio/cmd/resources/common.go:181]: can this be deleted?

**Review by Alevsk — CHANGES_REQUESTED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**